### PR TITLE
add title to truncated links

### PR DESCRIPTION
**Pull request by RabiaSajjad**

### CODE REVIEW

File: ckanext/canada/templates/package/base.html [hunk 1]
```diff
@@ -7,7 +7,6 @@
   {% set pkg_url = h.url_for(pkg.type ~ '.read', id=pkg.id if is_activity_archive else pkg.name, **({'activity_id': request.args['activity_id']} if 'activity_id' in request.args else {})) %}
   {% if g.action != 'new' and pkg %}
     {% set dataset = h.get_translated(pkg, 'title') %}
-    <li><a href="{{ pkg_url }}">{{ dataset|truncate(30) }}</a></li>
+    <li><a href="{{ pkg_url }}" title="{{ dataset }}">{{ dataset|truncate(30) }}</a></li>
```
Comment (JVickery-TBS) [inline ckanext/canada/templates/package/base.html:10]: @RabiaSajjad increase the `truncate` part here to more than `30`? or nah?

File: ckanext/canada/helpers.py [hunk 2]
```diff
@@ -861,6 +861,7 @@ def ckan_to_cdts_breadcrumbs(breadcrumb_content):
         anchor = breadcrumb.find('a')
         cdts_breadcrumbs.append({
             'title': breadcrumb.text if not anchor else anchor.text,
+            'acronym': '' if not anchor else anchor.get('title'),
```
Comment (JVickery-TBS) [inline ckanext/canada/helpers.py:864]: @RabiaSajjad cannot remember, I think I had problems with CDTS and the Jinja JSON filter with None/null values. Add empty string as default value for title?

```
anchor.get('title', '')
```

File: ckanext/canada/templates/admin/trash.html [hunk 3]
```diff
@@ -8,7 +8,7 @@
       <ul class="user-list list-group">
         {% for pkg in g.deleted_packages %}
           {% set title = pkg.title or pkg.name %}
-          <li class="list-group-item">{{ h.link_to(h.truncate(title, truncate_title), h.url_for(pkg.type + '.read', id=pkg.name)) }}</li>
+          <li class="list-group-item">{{ h.link_to(h.truncate(title, truncate_title), h.url_for(pkg.type + '.read', id=pkg.name), title=title) }}</li>
```
Comment (JVickery-TBS) [inline ckanext/canada/templates/admin/trash.html:11]: What is the value of `truncate_title` in this template, and does it need to be increased?

File: ckanext/canada/templates/package/resource_read.html [hunk 4]
```diff
@@ -31,6 +31,11 @@
   {% endif %}
 {% endblock %}
 
+{% block breadcrumb_content %}
+  {{ super.super() }}
+  <li class="active"><a href="" title="{{ h.resource_display_name(res) }}">{{ h.resource_display_name(res)|truncate(30) }}</a></li>
```
Comment (JVickery-TBS) [inline ckanext/canada/templates/package/resource_read.html:36]: I think the Resource names get pretty long, but we might still be able to support more than `30` chars here?

**Review by JVickery-TBS — CHANGES_REQUESTED** [review]

File: ckanext/canada/templates/package/base.html (hunk 1 above)
Comment (RabiaSajjad) [reply ckanext/canada/templates/package/base.html:10]: I can change it 80 so that it's the same everywhere.

File: ckanext/canada/helpers.py (hunk 2 above)
Comment (RabiaSajjad) [reply ckanext/canada/helpers.py:864]: I had no issues in testing, but I'll default to empty string in case I have missed anything.

File: ckanext/canada/templates/package/resource_read.html (hunk 4 above)
Comment (RabiaSajjad) [reply ckanext/canada/templates/package/resource_read.html:36]: I'll do 80 here as well.

File: ckanext/canada/templates/package/resource_read.html [hunk 5]
```diff
@@ -31,6 +31,11 @@
   {% endif %}
 {% endblock %}
 
+{% block breadcrumb_content %}
+  {{ super.super() }}
+  <li class="active"><a href="" title="{{ h.resource_display_name(res) }}">{{ h.resource_display_name(res)|truncate(80) }}</a></li>
```
Comment (wardi) [inline ckanext/canada/templates/package/resource_read.html:36]: AFAIK `<a>` tags without an href are bad for a11y. Are we allowed to add a title to another type of tag?

File: ckanext/canada/templates/snippets/cdts/header.html [hunk 6]
```diff
@@ -109,10 +109,6 @@
                 "subhref": {{ recombinant_menu_link('travela') | tojson }},
                 "subtext": {{ recombinant_menu_text('travela') | tojson }},
               },
-              {
-                "subhref": {{ recombinant_menu_link('qpnotes') | tojson }},
-                "subtext": {{ recombinant_menu_text('qpnotes') | tojson }},
-              },
```
Comment (wardi) [inline ckanext/canada/templates/snippets/cdts/header.html:115]: unrelated change?

Comment (RabiaSajjad) [reply ckanext/canada/templates/snippets/cdts/header.html:115]: yes, this is just removing a double entry of qpnotes in the menu

File: ckanext/canada/templates/package/resource_read.html (hunk 5 above)
Comment (JVickery-TBS) [reply ckanext/canada/templates/package/resource_read.html:36]: I think you can do javascript void into the href to fix the a11y complaints.

Comment (JVickery-TBS) [reply ckanext/canada/templates/package/resource_read.html:36]: Also put `tabindex="-1"` to prevent tabbing

**Review by JVickery-TBS — APPROVED** [review]

**Review by wardi — APPROVED** [review]
